package.json

```diff
@@ -86,6 +86,7 @@
     "electron-squirrel-startup": "^1.0.0",
     "keymirror": "0.1.1",
     "electron-context-menu": "^0.8.0",
+    "windows-shortcuts": "^0.1.4",
```

Comment:
i am not sure about using this library. it is bringing in some external shortcut.exe program.  i don't think we can trust that.  

other options
- build a native node module like: https://github.com/nadavbar/node-win-shortcut
- find another program that comes installed with 7 and greater that does same thing as shortcut.exe

i prefer the native node module myself.